Formula/s/sslyze.rb

```diff
@@ -33,38 +33,45 @@ class Sslyze < Formula
     end
   end
 
+  depends_on "pkg-config" => :build
   depends_on "pyinvoke" => :build
   depends_on "rust" => :build # for cryptography
-  depends_on "openssl@1.1"
+  depends_on "cffi"
+  depends_on "openssl@3"
```

Comment:
Note: if there is no problem with `openssl@3` (previous attempt at #134599), this can now be removed from `audit_exceptions/versioned_dependencies_conflicts_allowlist.json`.

Reply:
oh true, good call.